src/lib.rs

```diff
@@ -93,15 +93,24 @@ pub struct Config {
 
 #[derive(Clone, Debug)]
 pub struct Library {
+    /// Libraries specified by -l
     pub libs: Vec<String>,
+    /// Library search paths specified by -L
     pub link_paths: Vec<PathBuf>,
+    /// Library file paths specified without -l
+    pub link_files: Vec<PathBuf>,
+    /// Darwin frameworks specified by -framework
     pub frameworks: Vec<String>,
+    /// Darwin framework search paths specified by -F
     pub framework_paths: Vec<PathBuf>,
+    /// C/C++ header include paths specified by -I
     pub include_paths: Vec<PathBuf>,
+    /// Linker options specified by -Wl
     pub ld_args: Vec<Vec<String>>,
+    /// C/C++ definitions specified by -D
     pub defines: HashMap<String, Option<String>>,
+    /// Version specified by .pc file's Version field
     pub version: String,
-    _priv: (),
```

Comment:
This was here so outside code can't create values of this type. Allowing outside code to do that would make it a breaking API change to add fields, like you just did here. So let's better keep this :)

Reply:
We can just mark it as `#[non_exhaustive]` (unless that's not in 1.40? I think it is though)

Reply:
But the struct already is not `pub`

Reply:
It's public from what I can see: https://docs.rs/pkg-config/latest/pkg_config/struct.Library.html

Reply:
Ah right, it's just the `Library::new` constructor that is private.

Reply:
`#[non_exhaustive]` was [stabilized in 1.40](https://github.com/rust-lang/rust/pull/64639).